crates/common/chain/lean/src/gossip_request.rs

```diff
@@ -0,0 +1,7 @@
+use ream_consensus_lean::{block::SignedBlock, vote::SignedVote};
+
+#[derive(Debug, Clone)]
+pub enum LeanGossipRequest {
+    Block(SignedBlock),
+    Vote(SignedVote),
+}
```

Comment:
```suggestion
pub enum LeanP2PRequest {
    GossipBlock(SignedBlock),
    GossipVote(SignedVote),
}
```

Shouldn't we do this, then it always for sending other types of requests to the p2p service

Reply:
Then I think this enum should be defined under our p2p crate. I will move it while resolving this issue!